packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -314,28 +311,12 @@ function requestSignature(
   controller: AbortController,
   logger: Logger
 ): Promise<FetchResponse> {
-  if (!signersCircuitBreaker.has(service.url)) {
-    const options: Options = {
-      timeout: config.odisServices.timeoutMilliSeconds / 2,
-      resetTimeout: 15,
+  return parametrizedSignatureRequest(service.url, request, controller, logger).catch((e) => {
+    if (service.fallbackUrl) {
```

Comment:
It'd be great to log that the primary failed and we hit the fallback.

Reply:
Added